unison-core/src/Unison/Name.hs

```diff
@@ -120,6 +134,13 @@ stripNamePrefix prefix name =
   where
   mid = if toText prefix == "." then "" else "."
 
+-- suffixFrom Int builtin.Int.+ ==> Int.+
+-- suffixFrom Int Int.negate    ==> Int.negate
+suffixFrom :: Name -> Name -> Maybe Name
```

Comment:
What’s a concrete example of when to use this?

Reply:
Added comment on this - it's currently used as an implementation detail of `expandWildcardImports`, but it seemed general enough to include here.